machine-learning/tutorials/TransferLearningTF/Program.cs

```diff
@@ -60,53 +55,53 @@ private struct InceptionSettings
         // </SnippetInceptionSettings>
 
         // Build and train model
-        public static ITransformer ReuseAndTuneInceptionModel(MLContext mlContext, string dataLocation, string imagesFolder, string inputModelLocation, string outputModelLocation)
+        public static ITransformer GenerateModel(MLContext mlContext, string dataLocation, string imagesFolder, string inputModelLocation, string outputModelLocation)
```

Comment:
Suggestion:

Call the `GenerateModel` method `TrainAndEvaluateModel` or something like that since both are happening here.

Reply:
You might argue that evaluation is part of training :-) 

I think I'd like to leave this as generate, as the method returns the model ...